### ISSUE TITLE
Question regarding TenantAwareJob removal

### ISSUE THREAD

**Issue by AshotN** [hyn/multi-tenant#856]
Pardon my ignorance, I picked up a project recently that had been using version 5.2, so I've been trying to get it updated. One change between 5.3->5.4 is confusing me.

The documentation says, 

> The TenantAwareJob trait has been completely removed. Instead before dispatching any Job into the queue it will identify possible active tenants and remember/unserialize it when processing the job. You can still manually set the tenant website by giving the property website_id on the Job a value before dispatching it.

There is one usage of TenantAwareJob in the code-base, `use Queueable, TenantAwareJob;` my understanding is that TenantAwareJob is just no longer needed because Multi-tenant automatically handles jobs that are tenant related now.

But I don't want to break anything, so can anyone give me an ELI5 about the change

**Comment by Miguel-Serejo** [comment]
>can anyone give me an ELI5 about the change

You answered yourself:
>TenantAwareJob is just no longer needed because Multi-tenant automatically handles jobs that are tenant related now.


>I don't want to break anything

This is what testing is for.

**Comment by AshotN** [comment]
Alright cool, thanks.
